src/main.go

```diff
@@ -32,6 +32,7 @@ type BentoConfig struct {
 	MaxQueryParameters int    `envconfig:"BENTO_PUBLIC_MAX_QUERY_PARAMETERS"`
 	BentoPortalUrl     string `envconfig:"BENTO_PUBLIC_PORTAL_URL"`
 	Port               int    `envconfig:"INTERNAL_PORT" default:"8090"`
+	translated         bool   `envconfig:"TRANSLATED" default:"true"`
```

Comment:
`translated` should be `Translated` (capital T) so it can be exposed as a public variable to the rest of the code